extracted/vm/include/win/sqWin32.h

```diff
@@ -249,7 +249,7 @@ int reverse_image_words(unsigned int *dst, unsigned int *src,int depth, int widt
 */
 #define MAX_PATH_UTF8 (MAX_PATH*3)
 
-extern char  imageName [];       /* full path and name to image - UTF8 */
+extern char  _imageName [];       /* full path and name to image - UTF8 */
 extern WCHAR imageNameW[];       /* full path and name to image - UTF16 */
 extern char  imagePathA[];       /* full path to image - UTF8 */
 extern WCHAR imagePathW[];       /* full path to image - UTF16 */
```

Comment:
We should open a task for later to remove all these extern declarations.
We should have as few extern declarations as possible, and they should be really justified.
I don't think this is the case here...